app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -37,10 +37,6 @@ def as_json
 
   private
 
-    # An incomplete and temporary details hash that only currently includes
```

Comment:
The only problem is that it's no longer very clear that these tags are here for the purpose of generating email alerts. I think it would be helpful for future developers if that was made explicit somehow as it's not really documented anywhere.